test/api/importCSV/samples.go

```diff
@@ -0,0 +1,49 @@
+package importCSV
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+type TestCase struct {
+	fileName             string
+	ExpectedApplications []api.Application
+	ExpectedDependencies []api.Dependency
+}
+
+var (
+	Applications = []api.Application{
```

Comment:
Since more testcases (CSV files&expected resources to be created) are expected, I'd suggest move this directly into TestCase array.